manala.ohmyzsh/templates/themes/default.demo.j2

```diff
@@ -11,7 +11,7 @@ prompt_setup_manala(){
 
   FULL_HOSTNAME=`hostname -f`
 
-  base_prompt='%{$fg[magenta]%}%n%{$reset_color%}%{$fg[cyan]%}@%{$reset_color%}%{$fg[yellow]%}%{$FULL_HOSTNAME%}%{$reset_color%}%{$fg[red]%}:%{$reset_color%}%{$fg[cyan]%}%0~%{$reset_color%}%{$fg[red]%}|%{$reset_color%}'
+  base_prompt='%{$fg[magenta]%}%n%{$reset_color%}%{$fg[cyan]%}@%{$reset_color%}%{$fg[yellow]%}$FULL_HOSTNAME%{$reset_color%}%{$fg[red]%}:%{$reset_color%}%{$fg[cyan]%}%0~%{$reset_color%}%{$fg[red]%}|%{$reset_color%}'
```

Comment:
BTW, `$FULL_HOSTNAME` could be replaced by `%M` ideally, bit for whatever reason, it seems to not be the full hostname, whereas it should be.
: http://zsh.sourceforge.net/Doc/Release/Prompt-Expansion.html

Seems like it's the same issue as https://bugs.launchpad.net/ubuntu/+source/bash/+bug/1276796

Reply:
Yep, that's a long-standing well-known debian issue/feature :)